src/dbt/kipptaf/models/kippadb/intermediate/int_kippadb__persistence.sql

```diff
@@ -74,13 +152,64 @@ select
             e.actual_end_date is null and eis.ugrad_status in ('Graduated', 'Attending')
         then 1
         else 0
-    end as is_persisting_spring,
+    end as is_persisting_int,
+
+    case
+        when
+            date(r.academic_year + 1, 3, 31)
```

Comment:
I think you can refactor this calc to the roster CTE and avoid having to repeat it throughout the rest of the SELECT

Reply:
done